### PR TITLE
Whole Site Font Changes

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

It's better if you just look at the Netlify Preview 😉 

## Type of change

- [x] Yes

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: sass/custom/_paulus.scss [hunk 1]
```diff
@@ -114,15 +141,79 @@ $primary-color: #049cdb;
...
+  .vision {
+    margin-bottom: 24px;
+
+    h2 {
+      margin-bottom: 0;
+    }
+
+    h3 {
+      margin-top: .75em;
+    }
```
Comment (bramkragten) [inline sass/custom/_paulus.scss]: Can we make the `font-weight` of headers less? Like 400 or something.

Comment (zsarnett) [reply sass/custom/_paulus.scss]: Definitely down to do that. I do wish we had a font that had more settings for boldness. Aka a 500 and 600 difference. 

I will play with it

**Comment by bramkragten** [comment]
I think we should split this PR, one for the frontpage and one for the fonts etc.

**Comment by zsarnett** [comment]
Most of this is just the frontpage. The font changes are very small changes.  Color, font size and font weight. But I can definitely split them

**Comment by zsarnett** [comment]
This has now been split

**Comment by zsarnett** [comment]
One day we will do this. maybe :) probably not

**Comment by zsarnett** [comment]
Paulus stop blocking me 😄 This is better 😂

**Pull request closed without merging**